lib/index.js

```diff
@@ -1,5 +1,6 @@
 const Request = require('./request')
-const root = 'https://track.customer.io/api/v1'
+const trackRoot = 'https://track.customer.io/api/v1'
+const apiRoot = 'https://api.customer.io/v1'
```

Comment:
[So close!](https://learn.customer.io/api/#apibroadcast_trigger)

You just need to add `/api` to change the `apiRoot` to `'https://api.customer.io/v1/api'` here and in `test/index.js`. Beyond that the changes look spectacular. 🤩

Reply:
Oh wow, don't know how I missed that one. Fix incoming!